packages/web-components/src/components/breadcrumb/breadcrumb.ts

```diff
@@ -29,7 +29,7 @@ class CDSBreadcrumb extends LitElement {
    * Specify the size of the Breadcrumb. Currently
    * supports the following: `sm` & `md` (default: 'md')
    */
-  @property()
+  @property({ type: BREADCRUMB_SIZE, reflect: true, attribute: 'size' })
```

Comment:
```suggestion
  @property({ type: BREADCRUMB_SIZE, reflect: true })
```

Also no need to set the `attribute` here since the name is the same